src/components/Layout/Drawer.jsx

```diff
@@ -60,7 +59,6 @@ const Drawer = ({ open, setOpen }) => {
 
   return (
     <>
-      <Button onClick={toggleDrawer(true)}></Button>
       <DrawerSlide open={open} onClose={toggleDrawer(false)} anchor="right">
```

Comment:
```suggestion
      <DrawerSlide open={open} onClose={toggleDrawer} anchor="right">
```
여기서는 닫기에 대한 이벤트니까 무조건 setOpen(false)가 되겠지? 그럼 위처럼 핸들러를 등록하고,
핸들러는 매게변수 없이 아래처럼 하면 돼
```
const toggleDrawer = () => {
    setOpen(false);
  };
```
그리고 `toggleDrawer` 원래 형태처럼 () => () => 이어져 있는 건
커링함수라고해서 함수형 프로그래밍할때 쓰는 개념인데, 일단은 쓰지마